weather.py

```diff
@@ -68,7 +85,10 @@ def find_min(weather_data):
     Returns:
         The minium value and it's position in the list.
     """
-    pass
+    weather_data = [float(x) for x in weather_data]
+    min_num = min(weather_data)
+    min_index = weather_data.index(min_num)
+    return min_num, min_index
```

Comment:
This is a nice and clean approach. It's just missing a special check for cases where the input is empty, e.g. `if len(weather_data) == 0` or `if weather_data == []`.
This will fail the test which has 2 instances of the same minimum value, but it's technically a correct approach.